pkg/kubernetes/container/v1alpha1/container.go

```diff
@@ -193,9 +245,26 @@ func WithArguments(args []string) OptionFunc {
 	}
 }
 
-// WithArguments sets the command arguments of the container
-func (b *Builder) WithVolumeMounts(args []corev1.VolumeMount) *Builder {
-	WithVolumeMounts(args)(b.con)
+// WithVolumeMounts sets the command arguments of the container
+func (b *Builder) WithVolumeMounts(volumeMounts []corev1.VolumeMount) *Builder {
```

Comment:
Does this merge or result into duplicates

Reply:
Renamed it to WithVolumeMountsNew as it resets the field